### ISSUE TITLE
[Godot 4] Exported application cannot be launched on Android

### ISSUE THREAD

**Issue by DmitriySalnikov** [DmitriySalnikov/godot_qoi#6]
Porting to Godot 4 is complete (#5) . The library is successfully launched and works on windows, linux and macos, but crashes on android.
```
pid: 10419, tid: 10419, name: otengine.noname  >>> org.godotengine.noname <<<
      #00 pc 0000000002c1bdb0  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_android.so
      #01 pc 00000000000840b0  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_qoi.so (godot::StringName::init_bindings()+184) (BuildId: a97acf0c4e93adeac5cbf601f5bb13c30bc23d3b)
      #02 pc 000000000006f88c  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_qoi.so (godot::Variant::init_bindings()+104) (BuildId: a97acf0c4e93adeac5cbf601f5bb13c30bc23d3b)
      #03 pc 0000000000069590  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_qoi.so (godot::GDExtensionBinding::InitObject::init() const+104) (BuildId: a97acf0c4e93adeac5cbf601f5bb13c30bc23d3b)
      #04 pc 00000000000655e0  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_qoi.so (godot_qoi_library_init+84) (BuildId: a97acf0c4e93adeac5cbf601f5bb13c30bc23d3b)
      #05 pc 0000000002bc5b74  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_android.so
      #06 pc 0000000002bc72d0  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_android.so
      #07 pc 00000000028e8444  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_android.so
      #08 pc 00000000028e87bc  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_android.so
      #09 pc 00000000028eb988  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_android.so
      #10 pc 0000000002bcf8c4  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_android.so
      #11 pc 0000000002bd0640  /data/app/org.godotengine.noname-9hiNjlP0iOg5oq_HDZJS-w==/lib/arm64/libgodot_android.so
```

I don't know yet if the problem is in my library or in godot itself.

**Comment by HackLinjiuyue** [comment]
Maybe it is a good idea to wait for Godot 4 final release?

**Comment by DmitriySalnikov** [comment]
Well, maybe reporting the problems I've found will help fix them faster.
And also in this repository I finally updated the scripts for CI. In the future, I use these scripts in other repositories.

**Comment by DmitriySalnikov** [comment]
I finally found out that my library works without problems on version beta4. Beta5 and newer crashes with an error.
Here is the related issue https://github.com/godotengine/godot-cpp/issues/941

**Comment by DmitriySalnikov** [comment]
Fixed in one of the godot-cpp updates 7a6a7317cea5952b2a9a2e71fc1639997656076d
